lib/secretariat/tax.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Copyright Jan Krutisch
```

Comment:
I'm not sure how this works, or if I should have put my name in here? I personally don't care, but neither "signing in your name" nor "putting in my name here while I don't care about copyright at all" felt truly right.

Reply:
I'll come up with something along the lines of "Copyright Secretariat Team and Contributors, see CONTRIBUTORS.md" or something.